pyproject.toml

```diff
@@ -28,9 +28,9 @@ apscheduler = "3.7.0"
 asyncpg = "0.29.0"
 boto3 = "1.17.70"
 botocore = "1.20.71"
-flask = "2.2.5"
+flask = "^3.0.2"
```

Comment:
I think it shouldn't be here if it is installed from connexion extras, it can lead to an installation of incompatible version with connexion